jpype/config.py

```diff
@@ -0,0 +1,44 @@
+# *****************************************************************************
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#   See NOTICE file for details.
+#
+# *****************************************************************************
+
+# This files holds hints used to control behavior of the JVM
+# It is global variables rather than options to a function so that
+# they can be set a module at any time rather than requiring arguments
+# for a function.
+#
+# These options are to be treated as hints which may or may not be supported
+# in future versions.  Setting an unsupported option shall have no effect.
+# Variables shall not be removed even if deprecated so that conditions
+# based on these will be valid in all future versions.
+#
+
+onexit = True
```

Comment:
This style is plain and simple and could be replaced with something more complicated (if ever needed) in the future. So I completely agree with it.

Reply:
Is the name config reasonable?  Is there any other behaviors that should be configurable in this way besides shutdown?  

The basic idea is if there were multiple modules using jpype then the config module could gather all the requirements.  Most modules would use default so would never set these options.  This avoids having a huge number of options to startJVM and requiring the main from having to gather up all module requirements explicitely.